pkg/pool-manager/pool_test.go

```diff
@@ -82,6 +82,28 @@ var _ = Describe("Pool", func() {
 			table.Entry("Start: 02:FF:00:00:00:00  End: 02:00:00:00:00:00", "02:FF:00:00:00:00", "00:00:00:00:00:00", true),
 		)
 
+		table.DescribeTable("should check that a mac address in within the range", func(startMacAddr, endMacAddr string, testMacAddr string, shouldSucceed bool) {
+			startMacAddrHW, err := net.ParseMAC(startMacAddr)
+			Expect(err).ToNot(HaveOccurred())
+			endMacAddrHW, err := net.ParseMAC(endMacAddr)
+			Expect(err).ToNot(HaveOccurred())
+			testMacAddrHW, err := net.ParseMAC(testMacAddr)
+			Expect(err).ToNot(HaveOccurred())
+			isInRange := inRange(startMacAddrHW, endMacAddrHW, testMacAddrHW)
+			Expect(isInRange).To(Equal(shouldSucceed))
+		},
+			table.Entry("Start: 02:00:00:00:00:00  End: 03:00:00:00:00:00 Mac-Tested: 02:00:00:01:00:00", "02:00:00:00:00:00", "03:00:00:00:00:00", "02:00:00:01:00:00", true),
+			table.Entry("Start: 02:00:00:00:00:00  End: 03:00:00:00:00:00 Mac-Tested: 02:00:00:00:00:01", "02:00:00:00:00:00", "03:00:00:00:00:00", "02:00:00:00:00:01", true),
```

Comment:
how does the range is splitted ?
half half ?
what if the range has just one mac in case its legit ?
(or odd numbers? is it tested i guess)

Reply:
good question, but please note that range splitting is not the agenda of this PR. A separate PR will be published soon about it (and once merged then this PR will be rebased on top it - because tests can't pass otherwise)

I will state, that corner cases such as
- Where the range only consists of 1 MACs.
- Where one of the pod’s ranges given runs out.

are intentionally ignored as they are considered not relevant to the user story. This is also described in the [design document](https://docs.google.com/document/d/1oBgw5x3U3LHV2laKdNw9hNLJdPSbK9wjDJwihE8NusU/edit?usp=sharing).

Reply:
thanks for the explanation
not sure what is 
Where one of the pod’s ranges given runs out.
just out of curiosity
thanks

Reply:
when we assigned all the macs in one pod's pool.

Reply:
thx